solr/solr-ref-guide/modules/configuration-guide/pages/coreadmin-api.adoc

```diff
@@ -349,9 +349,19 @@ This parameter is required in v1, and part of the url in the v2 API.
 
 The `RENAME` action changes the name of a Solr core.
 
+[.dynamic-tabs]
```

Comment:
[+1] Thanks as always for your attention to the docs!

But....

[-1] I definitely see what you were going for in creating a different "Parameter" section for each API version.  It's tough otherwise to be clear about which parameter names are used in v1 vs. v2.  But in handling similar cases for other APIs, we've found a pattern that does this a bit more concisely without (I think) any loss of clarity.

Essentially what we've been doing is just to put the v1 and v2 parameter names in the same "list item" separated by a comma, and with the API version for each in parenthesis.  See the shards/shardNames parameter entry [here](https://solr.apache.org/guide/solr/latest/deployment-guide/collection-management.html#create-parameters) for an example.

Would you mind merging the "Parameters" sections here to be closer to that example?  (It's a bit of a hard thing to describe.  If I wasn't clear enough or you're not quite sure what I'm getting at, lmk I can push up a commit to show what I mean.)